packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -281,6 +301,11 @@ class SnapWPConfigManager {
 					cfg[ key ] = ( value.endsWith( '/' )
 						? value.slice( 0, -1 )
 						: value ) as unknown as T[ keyof T ];
+				} else if ( key === 'sitemap' ) {
```

Comment:
If @ayushnirwal doesn't tell you to move this logic somewhere else, then convert these to `if / continue` instead of `if / else / if ` and add a comment